appdatabase/migrations/sql/1719252625_nodeconfig_websocket.up.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE websocket_config (
+  enabled BOOLEAN DEFAULT false,
+  host VARCHAR NOT NULL DEFAULT "",
+  port UNSIGNED INT,
+  synthetic_id VARCHAR DEFAULT 'id' PRIMARY KEY
+) WITHOUT ROWID;
```

Comment:
1. I find it weird to store `websocket_config` separately from other `ApiConfig`. 
Maybe we should create a single `api_config` table and migrate all properties here?

-----

2. Do we really need to store this in DB?
This is something that's always set by client app. This value is never really taken from the database.

We discussed this in DM, but I want to share this with others as well. 

There is some confusion with `NodeConfig`:

- the initial goal is to start the `StatusNode` with some configuration. 
- we store it in the database to restore same config between launches
- still we apply some custom values from desktop and mobile
- still we apply some default values from status-go

Also, before `LoginAccount` endpoints we were passing the whole `NodeConfig` from the client. Now it's much better and it gives us opportunity not to do redundant things.

In a perfect world I think we should not store `NodeConfig` in the database directly. I mean we should store some fields, but "`NodeConfig`-structure agnostic". For example, like we store `websocket_config` in this PR.

Then we would do this on login:
1. take the default `NodeConfig` from status-go code
2. apply values from _settings_ database tables (like `websocket_config`, `settings`, etc.)
4. apply values from `LoginRequest`

This would create a clear path of building `NodeConfig`.

Reply:
For this PR, I did the websocket configuration in the same way that `http` and `ipc` had already been done. And I agree that all the saving of the config into the database is quite monstrous and I haven't found any scenarios where it would be useful, but maybe there are such scenarios when using status-go as a service. In any case, I suggest to start a separate ticket for refactoring of config storage

Reply:
Well, we can open a separate ticket. But I still think that in this case there's no need at all to add it to the database. We will not use it. 
I'd just keep it default in `defaultNodeConfig` and apply values from `LoginRequest`, if they're provided.

Reply:
Opened https://github.com/status-im/status-go/issues/5424

Reply:
I am in agreement with @igor-sirotin 
The only thing I am not 100% sure, is whether we allow clients to enable/disable websockets, in that case, it would be the only piece of config to store in the database.
But at this stage, I would not store anything in the database.

> take the default NodeConfig from status-go code
apply values from settings database tables (like websocket_config, settings, etc.)
apply values from LoginRequest

This is very important and the reason why we started moving away from the whole config passed to the client. We apply layers in order to get to the final config, and the client should specify as little as possible, and we should store as little as possible.

Reply:
Updated: i've removed the database